### PR TITLE
fixing a couple of typos

### PR DESCRIPTION
**Pull request by tmm88**
fixing a couple of typos

Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,22 +241,22 @@ We use [bunyan](https://github.com/trentm/node-bunyan) for logging:
...
   pipe anything that logs into `bunyan` e.g. `LOG_LEVEL=debug grunt test
   | node_modules/bunyan/bin/bunyan`
 
 
 ## Architecture
 
-In a nutshell the way the linter works is to take an add-on
-package, extract the metadata from the xpi (zip) format and then
+In a nutshell, the way the linter works is to take an add-on
+the package, extract the metadata from the xpi (zip) format and then
```
Comment (wagnerand) [inline README.md]: This reads odd.

File: README.md [hunk 2]
```diff
@@ -265,10 +265,10 @@ a set of rules that look for specific things.
 
 Rules get exported via a single function in a single file. A rule can
 have private functions it uses internally, but rule code should not depend
-on another rule file and each rule file should export one rule.
+on another rule file and each rule, file should export one rule.
```
Comment (wagnerand) [inline README.md]: I don't think that comma is right.

File: README.md [hunk 3]
```diff
@@ -265,10 +265,10 @@ a set of rules that look for specific things.
...
 
 Each rule function is passed data from the scanner in order to carry
-out the specific checks for that rule it returns a list of objects which
+out the specific checks for that rule, it returns a list of objects which
```
Comment (wagnerand) [inline README.md]: Can we make this a new sentence instead, please?

File: README.md (hunk 1 above)
Comment (EnTeQuAk) [reply README.md]: Yeah, that `the` feels wrong, the comma should be fine though.

**Comment by EnTeQuAk** [comment]
@tmm88 hey, did  you have any chance looking at the comments so that we can fix and merge this pull request?

File: README.md (hunk 1 above)
Comment (wagnerand) [reply README.md]: Yep.

**Comment by EnTeQuAk** [comment]
I'm closing this due to inactivity for now. If you have time to work on this at a later time I'd much appreciate if you could re-open this pull request and submit your changes.

Thanks a lot!

**Pull request closed without merging**